### DISCUSSION TITLE
Multiple angle fit

### DISCUSSION THREAD

**Discussion by quentintanguy** [PyEllips/pyElli#190] [General]
Is there a possibility to fit the dielectric parameters on a multiple angle psi_delta table?

**Comment by domna** [comment]
Hey @quentintanguy, yes, it is possible but you have to manually invoke the fitting routines. There is a new example which exactly explains the multiple angle fitting case. It is still in a PR state: https://github.com/PyEllips/pyElli/pull/186. However, it will be merged soon (but of course you can already copy the ipynb from there and use it as a basis for your own code).

**Reply by domna** [reply]
Here is the merged example in our docs: https://pyelli.readthedocs.io/en/latest/auto_examples/plot_03_custom_fitting.html#sphx-glr-auto-examples-plot-03-custom-fitting-py

and the corresponding ipynb: https://github.com/PyEllips/pyElli/tree/master/examples/Custom%20fitting
